### DISCUSSION TITLE
Contributor Call: 29 October 2022

### DISCUSSION THREAD

**Discussion by l3xander** [freesewing/freesewing#2973] [Support] [answered]
You are all invited to the FreeSewing Contributor Call.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This issue will be updated after the call with notes and to-dos.

### About
FreeSewing's contributor call is an informal hangout/chat/meeting that is open to all FreeSewing contributors and users alike.
The call is organized/hosted by @tangerineshark and @kaermorhan.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

**Details**
Location: https://discord.freesewing.org (meeting room: 🔊`voice chat`)
Date: 29 October 2022 4pm UTC, which can be found in your local time zone [here](https://savvytime.com/converter/utc-to-belgium-antwerpen-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/oct-29-2022/4pm).
Or check the bot in Discord! `#meetings`
The meetings will always be at 6pm, Antwerp time.

### Agenda
- v3 docs are published on https://freesewing.dev
- v2 docs remain available on https://v2.freesewing.dev
- Strapi will remain available on https://posts.freesewing.org until we migrate to v3 frontend but will not be migrated
- Sanity and https://cms.freesewing.org (and can we afford it)
- Measurements and their names - @nchilada 

> Notes will be added after the call.

**Comment by KaerMorhan** [comment, marked as answer]
**Notes**

Participants:
@tangerineshark 
@lexander2002 
@KaerMorhan 
@joostdecock 
@bobgeorgethe3rd 
@BenJamesBen 
@SeaZeeZee 
@woutervdub 
@nataliasayang 

✨ **Joost**
V3 documentation was [published](https://freesewing.dev).
V2 documentation is still available on https://v2.freesewing.dev/
_Benjamin’s feedback: I’m planning to review V3 docs._

Strapi will remain available on https://posts.freesewing.org until we migrate to v3 frontend but will not be migrated. Joost contacted [Sanity](https://www.sanity.io) to discuss pricing and received an answer: FS will be on their non-profit plan.
Strapi was moved to Sanity: https://cms.freesewing.org/ (an email is needed to access it). Now showcase images have more display options.
Translations will be split now between Sanity and Crowdin. Help with showcase and blogposts is welcome.
_Karen’s feedback: How to do showcase now?
Joost: I tried using Sanity for another project and I am sure that we can make it work.
Karen: How will newsletters work now with some email changes?
Joost: We will still have Strapi redirection but please don’t depend on it._

I’m working on a backend now so it would be possible to have a local version of FS that wouldn’t require a backend. It will be moved to SQLite.
_Zwaluw: what is backend?
Joost: it’s an API that stores user profile information. It is also a way to talk to the database. It is ReST API for FS._

✨ **Karen**
I wanted to say sorry for not catching a heated discussion sooner in one of the chats.
[Community standards](https://freesewing.org/docs/various/community-standards/) | [Code of conduct](https://freesewing.dev/guides/code-of-conduct)
_Joost: I muted FS server a while ago so if you need some moderation or an answer from my side, you can ping me.
Zwaluw: I would like to be a moderator._

Also: Zwaluw, how about an interview?
_Zwaluw: that would be wonderful._

✨ **Zwaluw**
Can I take Jane off of your shoulders somehow, Joost? It’s an open pull-request now.
_Joost: every new pattern will not be available until we premiere V3. It is done through monorepo and GitHub. Benjamin can also help you with testing._

✨ **What I'm working on right now**

**Natalia**
I’m sewing a dinosaur onesie.
_Wouter: we definitely need that pattern._

**Zwaluw**
I’m doing my 18th century studies and thinking about more patterns: underwear, stays/corset etc.

**Karen**
I’m working on a Hugo rn.
_Natalia: Have you considered putting spikes on it so it looks like a dinosaur?
Joost: Maybe we need a new rule – no spikes, no showcase._

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions
